docs/error_messages.md

```diff
@@ -0,0 +1,368 @@
+# Error Counters
+
+Issues found in the input files are categorized under "counters". Counters
+aggregate the issues, and provide a high-level overview of what went wrong.
+
+Error counters with "**Suffix Description:**" are prefix counters that are
+logged with a suffix detailing the subject of the error. The For example,
+`Resolution_UnresolvedExternalId_` will be suffixed with the ID that could not 
+be resolved. The "**Suffix Description:**" field describes the nature of the
+suffix for these counters.
+
+## MCF_UnenclosedComplexValue
+
+**Description:** Complex value was not enclosed in brackets `[]` in the MCF.
+
+### Suggested User Actions
+
+1. Check complex values in your MCF and make sure they are enclosed in brackets
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## MCF_MalformedComplexValueParts
+
+**Description:** Complex value had less than 2 or more than 3 parts
+
+### Suggested User Actions
+
+1. Check that all of your complex values have either 2 or 3 parts
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## MCF_QuantityMalformedValue
+
+**Description:** In a complex value with 2 parts, the part that was expected to be a number was not a number
+
+### Suggested User Actions
+
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## MCF_InvalidLatitude
+
+**Description:** Invalid latitude part in complex value; latitude must be decimal degrees with an optional N/S suffix
+
+### Suggested User Actions
+
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## MCF_InvalidLongitude
+
+**Description:** Invalid longitude part in complex value; longitude must be decimal degrees with an optional E/W suffix
+
+### Suggested User Actions
+
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## MCF_QuantityRangeMalformedValues
+
+**Description:** An unexpected part was found in the complex value in MCF, error message will specify the type of issue
+
+### Suggested User Actions
+
+1. Either the start or end components are wrong, or at least one of the components have to be a number. Check the error message in `report.json`.
+1. Refer to the [documentation of valid complex values](complex_values.md)
+
+## Resolution_UnresolvedExternalId_
+
+**Description:** External ID reference could not be resolved.
+
+**Suffix Description:** External ID reference could not be resolved.
```

Comment:
We shouldn't duplicate this?

Instead, say this refers to an external ID property whose value isn't resolved.